src/dispatch/static/dispatch/src/case/ResourcesTab.vue

```diff
@@ -65,23 +65,75 @@
       </v-list-item>
       <v-divider />
     </span>
+    <span
+      v-if="
+        (!ticket && ticketPluginEnabled) ||
+        (!groups && groupPluginEnabled) ||
+        (!conversation && conversationPluginEnabled) ||
+        (!storage && storagePluginEnabled) ||
+        (!documents && documentPluginEnabled)
+      "
+    >
+      <v-list-item @click="createAllResources()">
+        <v-list-item-content>
+          <v-list-item-title>Recreate Missing Resources</v-list-item-title>
+          <v-list-item-subtitle
+            >Initiate a retry for creating any missing or unsuccesfully created
+            resource(s).</v-list-item-subtitle
+          >
+        </v-list-item-content>
+        <v-list-item-action>
+          <v-list-item-icon>
+            <v-icon>refresh</v-icon>
+          </v-list-item-icon>
+        </v-list-item-action>
+      </v-list-item>
+    </span>
   </v-list>
 </template>
 
 <script>
+import { mapActions } from "vuex"
 import { mapFields } from "vuex-map-fields"
 
 export default {
   name: "CaseResourcesTab",
 
+  data() {
+    return {
+      ticketPluginEnabled: false,
+      groupPluginEnabled: false,
+      conversationPluginEnabled: false,
+      storagePluginEnabled: false,
+      documentPluginEnabled: false,
+    }
+  },
+
   computed: {
     ...mapFields("case_management", [
+      "selected",
       "selected.documents",
       "selected.groups",
       "selected.storage",
       "selected.ticket",
       "selected.conversation",
     ]),
   },
+
+  async mounted() {
+    this.ticketPluginEnabled = await this.isPluginEnabled("ticket")
+    this.groupPluginEnabled = await this.isPluginEnabled("group")
+    this.conversationPluginEnabled =
+      (await this.isPluginEnabled("conversation")) &&
+      this.selected.case_type &&
+      this.selected.case_type.conversation_target
+    this.storagePluginEnabled = await this.isPluginEnabled("storage")
+    this.documentPluginEnabled = await this.isPluginEnabled("document")
+    console.log(this)
```

Comment:
Would it be better to get all enabled plugins for this project once and then check the list to see if the type is on the list rather than making 5 separate API calls?

Reply:
I'm planning to add a feature that allows users to select which resource to add. 
For example, only creating a storage resource.

Reply:
Still, one could retrieve the enabled plugin list once on load and check against the list as needed.

Reply:
Done.